examples/http-protocol-versions/verify.sh

```diff
@@ -21,9 +21,16 @@ for path in "${path_list[@]}"; do
     curl -svk -o /dev/null --http2 --http2-prior-knowledge "https://localhost:10001/${path}" 2>&1 | grep "HTTP/2 200"
 done
 
+run_log "Build simple HTTP/3 client container"
+pushd ./http3-client
+docker build -t h3client:test .
```

Comment:
hi @su225 thanks for working on this

the way that the examples work means that this should not be necessary

all of the builds should used docker-compose

if you really need multiple docker/compose builds and you can specify that multiple builds are brought up for the verification test (if required)  using the `PATHS` var (see cors example)

if you need to bring up the env manually you can also do that with the `MANUAL` env var (see eg `websocket` example)

there is  a `README` that goes through how the example verification works here - https://github.com/envoyproxy/envoy/blob/main/examples/DEVELOPER.md

Reply:
Thank you @phlax . I'll refactor these parts to conform to other tests. I wanted to package my custom HTTP/3 client in the docker container and then run it to verify HTTP/3 -> HTTP(*) works.

Reply:
Using `MANUAL=true` as it also includes generating certificates on the fly. Anyways, refactored HTTP/3 client to use docker-compose.